lib/tasks/organizations.rake

```diff
@@ -1,14 +1,9 @@
-namespace :organizations do
-  task :setup, [:name, :contact_email, :book_slug, :logo_url] => :environment do |t, args|
-    args.with_defaults(logo_url: "#{Rails.configuration.base_url}/logo-alt-large.png")
+logger = Mumukit::Nuntius::Logger
 
-    organization = Organization.create! name: args[:name],
-                                        contact_email: args[:contact_email],
-                                        book: Book.find_by(slug: args[:book_slug]),
-                                        logo_url: args[:logo_url],
-                                        private: true
-    organization.switch!
+namespace :organizations do
+  task listen: :environment do
+    logger.info 'Listening to organizations'
 
-    puts 'Congrats. A new organization has been created'
+    Mumukit::Nuntius::EventConsumer.start 'office'
```

Comment:
@flbulgarelli Sorry, I mean, the job `events` and the task `events.rake` already exist (listening to 'atheneum'). Should I delete `organizations.rake` then?